ui/pages/[page].js

```diff
@@ -2,16 +2,17 @@ import { parse } from 'marked';
 import DOMPurify from 'isomorphic-dompurify';
 import { fromMarkdown } from 'mdast-util-from-markdown';
 import { getPages } from '../helpers/api';
-import { Page, TableOfContents } from '../components';
+import { Page, TableOfContents, CookiesTable } from '../components';
 
-const StaticPage = ({ content, showToc, title, parsed }) => {
+const StaticPage = ({ content, showToc, title, parsed, page }) => {
   return (
     <Page title={title}>
       <div className="nhsuk-grid-row">
         {showToc && <TableOfContents content={parsed} />}
         <div className="nhsuk-grid-column-two-thirds">
           <h1>{title}</h1>
           <div dangerouslySetInnerHTML={{ __html: content }} />
+          {page === 'cookie-policy' && <CookiesTable />}
```

Comment:
Would it be better to actually make a page?

Reply:
Or is it still loading _some_ content from CKAN?

Reply:
I missed this, but yes, there are a fair few words on CKAN still